voltdb-stored-procedures-maven-quickstart/src/main/resources/archetype-resources/src/integration-test/java/TestBase.java

```diff
@@ -37,7 +37,7 @@ public class TestBase {
     }
 
     public String getImageVersion() {
-        return props.getProperty("voltdb.image.version", "14.3.1");
+        return props.getProperty("voltdb.image.version", "14.3.0");
```

Comment:
I set the default back one version just so it's different from the latest version we're setting in the pom. Maven uses resource filtering to take the property value from the pom and populate the test.properties resource file, which TestBase then loads. It has to be a published image for the test to succeed, which is why it's not yet set to 15.0.0.